packages/client/tests/functional/methods/count/tests.ts

```diff
@@ -117,17 +115,17 @@ testMatrix.setupTestSuite((_suiteConfig, _suiteMeta, clientMeta) => {
       Invalid \`prisma.user.count()\` invocation in
       /client/tests/functional/methods/count/tests.ts:0:0
 
-        XX })
+         XX })
         XX 
-        XX testIf(clientMeta.runtime !== 'edge' && getQueryEngineProtocol() !== 'json')('bad prop', async () => {
+        XX testIf(clientMeta.runtime !== 'edge')('bad prop', async () => {
       → XX   const err = prisma.user.count({
                 select: {
                   _count: {
                     select: {
-              ?       _all?: true,
-              ?       email?: true,
-              ?       age?: true,
-              ?       name?: true,
+                      _all: true,
+                      email: true,
+                      age: true,
+                      name: true,
```

Comment:
why are these now marked as required?

Reply:
These fields are already passed in by the user, so they are not suggestions, they pretty-print of the input.